### PR TITLE
getElapsedTime() in TREE.Clock returns whole running time and not elapsed time from previous start.

### PR DESCRIPTION
**Pull request by Verokster**
getElapsedTime() function returns incorrect value after stop() and start(), It returns whole working/running time from first run.
I adde this.elapsedTime = 0 inside stop() function after this.getElapsedTime() line, so after next start() we can get correct elapsed time from last start()

Perhaps it's incorrect and this suggestion need new function, so Clock.js will have two functions:
- first one will return whole running time (as it is now)
- second will return elapsed time from last start

### CODE REVIEW

**Comment by Verokster** [comment]
Also, I thought to remove getElapsedTime() from "stop" function as there is no needs to update this.elapsedTime, But getElapsedTime() updates this.oldTime at "stop" action, so after "stop" action we have a valid startTime and oldTime (as stop time), and we can calculate timespan of previous running

**Comment by mrdoob** [comment]
Shouldn't this go in `start()` instead?

File: src/core/Clock.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ Clock.prototype = {
 	stop: function () {
 
 		this.getElapsedTime();
+		this.elapsedTime = 0;
```
Comment (mrdoob) [inline src/core/Clock.js]: Shouldn't this go in `start()` instead?

Comment (Verokster) [inline src/core/Clock.js]: Yes, you are right :D.
Moved in second commit. Review it please

**Comment by mrdoob** [comment]
Thanks!

**Comment by Verokster** [comment]
You too!

**Pull request merged**